packages/mobile-top-nav/src/css/media-menu.js

```diff
@@ -0,0 +1,69 @@
+import { css } from 'lit-element';
+
+export default () => {
```

Comment:
How come you have all of these css blocks as separate files instead of part of their component?

Reply:
I still like the idea of separating concerns. Going this route, if there are general styles that you want to include on multiple components, you could then

```
static get styles() {
  return [baseCss, cascadeCss];
}
```

Reply:
Hrm I think this adds an unnecessary layer of indirection and is not really a separation of concerns. The WebComponent is a self-contained thing: markup, CSS, and logic. One of its huge benefits is to be able to work on it in isolation without jumping around through different files. If we split it up, we're now maintaining two parallel directories of CSS and Components and that is going to be a bear to manage.

I also don't think you should import external CSS's into your component. The style in your component should be completely self-contained. You quickly begin to run into the same problem with old school CSS where you become afraid to touch your shared stylesheet because you don't know what components you're going to affect. 

If you have customization points, they should be done with CSS Variables so the component is only exposing what it knows about.